.github/workflows/upload-bottles.yml

```
@@ -28,6 +28,11 @@ jobs:
           echo sha1='${{github.event.head_commit.id}}'
           echo '${{github.event.head_commit.message}}'
           pr="$(echo '${{github.event.head_commit.message}}' | sed 's/^.*#\([0-9]*\).*/\1/;q')"
+          if [[ $(curl -fs -H 'Accept: application/vnd.github.antiope-preview+json' https://api.github.com/repos/${{github.repository}}/pulls/$pr \
+            | jq '.labels[] | select(.name == "skip upload")') ]]; then
```

```suggestion
            | jq '.labels[] | select(.name == "ci-skip-upload")') ]]; then
```
Bike shedding here, but how about `ci-skip-upload`. I'm thinking a `ci-` prefix for any labels that have an actual effect on CI, and hyphens instead of spaces to make shell usage easier without having to quote the spaces `gh pr create -lci-skip-upload`

---

Shall we rename `ci_error` to `ci-error`?

---

Homebrew/core uses either spaces or hyphens, and I prefer hyphens, so let's go with hyphens. Feel free to rename tags with underscores to hyphens if you like.